src/analyzer/protocol/ldap/ldap.spicy

```diff
@@ -641,13 +641,16 @@ public function utf16_guid_to_hex_repr(bts: bytes): string {
         return "GUID_FORMAT_FAILED";
     }
 
+    local guid_parts: vector<vector<uint8>> = [[3, 2, 1, 0], [5, 4], [7, 6], [8, 9], [10, 11, 12, 13, 14, 15]];
+    local guid_breaks: set<uint8> = [0, 4, 6, 9];
+
     local ret = "";
-    for (i in [[3, 2, 1, 0], [5, 4], [7, 6], [8, 9], [10, 11, 12, 13, 14, 15]]) {
+    for (i in guid_parts) {
```

Comment:
The temporary we previously created here already lived for the duration of the function, so this should behave identically, but could be less clear. Similar for `sid_parts` and `reversed` below. My hunch would be to keep the previous form.